packages/sdk/package.json

```diff
@@ -2,25 +2,58 @@
   "name": "@namada/sdk",
   "version": "0.2.4",
   "description": "Namada SDK package",
-  "main": "dist/index.js",
-  "types": "dist/index.d.ts",
+  "exports": {
+    "./web": {
+      "import": "./dist/web/sdk/src/indexWeb.js",
+      "types": "./dist/web/sdk/src/indexWeb.d.ts"
+    },
+    "./web-init": {
```

Comment:
It's done like that because otherwise there was some problem in unit tests as we build wasm for tests as nodejs target. When we couple init with index, TS will complain that module produced by wasm pack does not export some init functions. Something like that :D So it's not really "needed" outside of tests. Maybe we could improve that in future by running tests in browser or sth?